secret_settings.py

```diff
@@ -21,5 +21,11 @@
             secrets = json.load(f)
     else:
         raise FileNotFoundError("secrets.json file not found. Please refer to the Cosmos website wiki.")
+elif sys.platform.startswith("darwin"):
+    if os.path.exists("/etc/secrets.json"):
+        with open("/etc/secrets.json", "r") as f:
+            secrets = json.load(f)
+    else:
+        raise FileNotFoundError("secrets.json file not found. Please refer to the Cosmos website wiki.")
```

Comment:
There is no difference in the file handling for macos compared to linux it seems, so it is cleaner to just change the if statement on line 9 to also hold for macos

Reply:
will do!